.htaccess

```diff
@@ -66,10 +66,6 @@ RewriteRule (.*) public/$1 [L]
   # JavaScript
     ExpiresByType application/javascript                "access plus 1 year"
 
-  # Manifest files
-    ExpiresByType application/x-web-app-manifest+json   "access plus 0 seconds"
```

Comment:
Why remove this?

Reply:
Never mind, the corect type is `application/manifest+json`

Reply:
My reasonning was to let apache handle this for static files.